controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3623,4 +3623,7 @@ OperationFailedRuntimeException capabilityAlreadyRegisteredInContext(String capa
 
     @Message(id = 475, value = "Value for attribute '%s' is invalid.")
     OperationFailedException invalidAttributeValue(String attributeName);
+
+    @Message(id = 476, value = "Invalid attribute(s) '%s' .")
+    OperationFailedException invalidAttribute(ModelNode node);
```

Comment:
I'm not sure that the message is meaningful enough... What makes the attribute invalid? It would be more informational to put the keys that are still present in the node...

Reply:
Thanks, I have added more information into the message and made it more clear.